src/components/input-group/daikin-input-group.ts

```diff
@@ -78,39 +84,46 @@ export class DaikinInputGroup extends LitElement {
       --input-group-border-color-error: ${unsafeCSS(colorFeedbackNegative)};
 
       display: block;
-      width: max-content;
+      width: 100%;
     }
   `;
 
   /**
-   * Label text to place at the top of the field
+   * Label text displayed at the top of the field.
    */
   @property({ type: String })
-  label?: string;
+  label = "";
 
   /**
-   * Helper text to place at the bottom of the field. In error conditions, this text is hidden.
+   * Helper text displayed at the top of the field.
+   * If `error` is specified and `disabled` is `false`, that takes precedence and helper text is not displayed.
    */
   @property({ type: String })
-  helper?: string;
+  helper = "";
 
   /**
-   * Whether the field is disabled. Reflected in the `disabled` property of the input in the slot.
+   * Error text displayed at the right of the label.
+   * Ignored if `disabled` is `true`.
+   * If `true`, an additional star mark will be displayed at the right of the label text.
+   * Reflected in `required` attribute of the input control in the slot.
    */
-  @property({ type: Boolean, reflect: true })
-  disabled = false;
+  @property({ type: String, reflect: true })
+  required = "";
```

Comment:
If you set `reflect: true` and use an empty string as the default, it will probably add the attribute and cause confusion.

```suggestion
  @property({ type: String, reflect: true })
  required: string | null = null;
```